app/models/puzzle.rb

```diff
@@ -2,4 +2,6 @@ class Puzzle < ApplicationRecord
   enum :answer, ruby: 0, rails: 1
   enum :state, { approved: 0, rejected: 1, pending: 2, archived: 3 }
   has_many :answers
+
+  scope :archived, -> { Puzzle.where(state: :archived).order(sent_at: :desc) }
```

Comment:
Adding the class, doesn’t add a rails deprecation warning?

```suggestion
  scope :archived, -> { where(state: :archived).order(sent_at: :desc) }
```

Reply:
If it does, it doesn't show up in my logs nor in the console, but I liked the change in any event